software/util/buildsite.py

```diff
@@ -277,8 +290,8 @@ def templateRender(template_path, extra_vars=None, template_instance=None):
 def checkFilePath(path):
     if not path in CHECKEDPATHS:
         CHECKEDPATHS.append(path)
-        if not path.startswith('/'):
-            path = os.getcwd() + "/" + path
+        if not os.path.isabs(path):
```

Comment:
this is not necessary, os.path.join() already discards previous segments if a path element is absolute.